tutorials/transparent-proxy-and-filtering-on-k8s-with-initializers/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Transparent Proxy and Filtering on Kubernetes with Initializers
+description: Learn how to run a transparent proxy on Kubernetes to filter and intercept traffic out of your deployments using custom deployment initializers.
+author: danisla
+tags: Kubernetes mitmproxy proxy
+date_published: 2017-09-23
+---
+
+Dan Isla | Google Cloud Solution Architect | Google
+
+This is a follow-on tutorial to the [Transparent Proxy and Filtering on K8S](https://cloud.google.com/community/tutorials/transparent-proxy-and-filtering-on-k8s) tutorial and shows how to simplify the application of transparent proxy for existing deployments using a Deployment Initializer. Initializers are one of the [Dynamic Admission Control](https://kubernetes.io/docs/admin/extensible-admission-controllers/) features of Kubernetes and available as an alpha feature as of Kubernetes 1.7.
+
+This tutorial uses the [tproxy-initializer](https://github.com/danisla/kubernetes-tproxy/tree/master/cmd/tproxy-initializer) Kubernetes Initializer to inject the sidecar InitContainer, ConfigMap and environment variables into a deployment when the annotation `"initializer.kubernetes.io/tproxy": "true"` is present. This tutorial also demonstrates how to deploy the tproxy Helm chart with the optional [Role Based Access Control](https://kubernetes.io/docs/admin/authorization/rbac/) support.
```

Comment:
Maybe add a link to tproxy Helm chart, unless your audience will typically already know what that is.